src/pages/course-page.js

```diff
@@ -104,6 +111,8 @@ const CoursePage = (props) => {
           workloadProgress={review.rating.workload}
           enjoymentProgress={review.rating.enjoyment}
           difficultyProgress={review.rating.difficulty}
+          author={checkAuthor(review)}
```

Comment:
actually, since it's a one liner, u could just chuck it here and no need to extract to a new function